miniapp/USERGUIDE.md

```diff
@@ -153,6 +161,8 @@ class MiniAppActivity : Activity(), CoroutineScope {
 }
 ```
 
+`MiniAppDisplay.navigateBackward` and `MiniAppDisplay.navigateForward` facilitates the navigation inside a mini app if the history stack is available in it. A common usage pattern could be to link it up to the Android Back Key navigation.
```

Comment:
I think we should have a code example of this. Maybe this can be moved to the `Advanced` section and then add a code sample similar to what's in the Sample App:

```
override fun onBackPressed() {
    if(!miniAppDisplay.navigateBackward()) {
        super.onBackPressed()
    }
}
```

Reply:
Okay. This could be added too.